src/FunctionLoader.cs

```diff
@@ -57,11 +58,23 @@ internal static void SetupWellKnownPaths(FunctionLoadRequest request)
             var appLevelModulesPath = Path.Join(FunctionAppRootPath, "Modules");
             var workerLevelModulesPath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             FunctionModulePath = $"{appLevelModulesPath}{Path.PathSeparator}{workerLevelModulesPath}";
-
+            AddLatestAzModulesPath();
             // Resolve the FunctionApp profile path
             var options = new EnumerationOptions { MatchCasing = MatchCasing.CaseInsensitive };
             var profiles = Directory.EnumerateFiles(FunctionAppRootPath, "profile.ps1", options);
             FunctionAppProfilePath = profiles.FirstOrDefault();
         }
+
+        /// <summary>
+        /// Adds the latest AzModules path to FunctionModulePath
+        /// </summary>
+        private static void AddLatestAzModulesPath()
+        {
+            if (!string.IsNullOrWhiteSpace(FunctionModulePath)
+                && !string.IsNullOrWhiteSpace(latestAzModulePath))
```

Comment:
Neither the `FunctionModulePath` nor the `latestAzModulePath` can be null at the moment. So you don't need to check for these.

Reply:
This is just for safety, but I can remove it